docs/changes/latest.inc

```diff
@@ -40,6 +40,9 @@ Enhancements
 - Add support for nilearn computed masks (``compute_epi_mask``, ``compute_brain_mask``, ``compute_background_mask``,
   ``fetch_icbm152_brain_gm_mask``) (:gh:`175` by `Fede Raimondo`_).
 
+- Add fMRIprep brain masks to the datagrabber patterns for all datagrabbers in the aomic sub-package
```

Comment:
Just to follow the naming convention: Add fMRIPrep ...

Reply:
oops yeah always get that wrong